gatsby-node.js

```diff
@@ -45,36 +45,42 @@ exports.createPages = ({ graphql, actions }) => {
     const posts = result.data.allMarkdownRemark.edges;
 
     posts.forEach((post, index) => {
-      if (post.node.fields.sourceInstanceName === 'blog') {
-        const previous =
-          index === posts.length - 1 ? null : posts[index + 1].node;
-        const next = index === 0 ? null : posts[index - 1].node;
-        const { sourceInstanceName, slug } = post.node.fields;
-
-        console.log(`Create pages /${sourceInstanceName}${slug} from ${slug}`);
-        console.log('------------------------------');
-        createPage({
-          path: `/${sourceInstanceName}${slug}`,
-          component: blogPost,
-          context: {
-            slug: post.node.fields.slug,
-            previous,
-            next,
-          },
-        });
-      } else if (post.node.fields.sourceInstanceName === 'platform') {
-        const { sourceInstanceName, slug } = post.node.fields;
-
-        console.log(`Create pages /${sourceInstanceName}${slug} from ${slug}`);
-        console.log('------------------------------');
-        createPage({
-          path: `/${sourceInstanceName}${slug}`,
-          component: platform,
-          context: {
-            slug: post.node.fields.slug,
-            tags: post.node.frontmatter.tags || [],
-          },
-        });
+      // Don't create a page for any markdown file with a title of Aside.
+      // MD files with Aside in the title are used exclusively for creating links.
+      if (post.node.frontmatter.title !== 'Aside') {
```

Comment:
This might be better done as a separate frontmatter field, like isAside or could be more generically done as a 'type' field or such rather than keying off a 'special' title. If for some reason we ever did localize this kind of thing trips up localizers.

Reply:
changes made to aside.md. It now has an isAside field which is checked for when creating a page. I no have the condition set for onCreateNode so aside.md receives a field parameter. Added an extra filter on the query to prevent it from displaying on the /platforms page